msg/tools/urtps_bridge_topics.yaml

```diff
@@ -28,31 +28,31 @@
 rtps:
   # topic ID 1
   - msg:     debug_array
-    receive: true
+    # receive: true
```

Comment:
Was this just for testing? Can you restore the file?

Reply:
Yes, correct. The file has been restored.